server/views/pages/scan/scan-barcode-result.njk

```diff
@@ -40,6 +40,7 @@
       {% if form.errorCode.code == 'FURTHER_CHECKS_NEEDED' %}
         {% include "partials/scan/scan-barcode-result-further-checks-needed.njk" %}
       {% endif %}
+      <span id="scan-result-type" data-value="{{ 'READY_TO_DELIVER' if not form.errorCode.code else form.errorCode.code }}" />
```

Comment:
The new element will be one of:
* `<span id="scan-result-type" data-value="READY_TO_DELIVER" />`
* `<span id="scan-result-type" data-value="DUPLICATE" />`
* `<span id="scan-result-type" data-value="RANDOM_CHECK" />`
* `<span id="scan-result-type" data-value="EXPIRED" />`
* `<span id="scan-result-type" data-value="NOT_FOUND" />`
* `<span id="scan-result-type" data-value="CANNOT_ENTER_BARCODE" />`
* `<span id="scan-result-type" data-value="FURTHER_CHECKS_NEEDED" />`
as proved by the unit test 👇